README.md

```diff
@@ -1,14 +1,14 @@
-# CLAW Vagrant Playbook
+# Islansdora 8 Vagrant Playbook
```

Comment:
Got an extra s in there

Reply:
Well, it's still technically a de-CLAW... 😛